cypress/obj/timeLoggingPageObject.js

```diff
@@ -0,0 +1,23 @@
+class TimeLoggingPage {
+    calendarShouldbeVisible(){
+        cy.get('.calendar').should('be.visible')
```

Comment:
Do not do assertions in here

Reply:
ok, but my assert is in test, it just encapsulated :P